src/delta.rs

```diff
@@ -90,11 +114,20 @@ where
                 handle_hunk_meta_line(&mut painter, &line, config)?;
                 continue;
             }
+        } else if source == Source::DiffUnified && line.starts_with("Only in ") {
+            state = State::FileMeta;
```

Comment:
I want to avoid generating a new state, and since this section should behave like a new file block, I used the same state
Let me know if you'd rather have a new state for it

Reply:
Reusing the same state here makes sense to me.